app/mailers/rrd_completed_mailer.rb

```diff
@@ -5,7 +5,7 @@ def build(submission)
     @submission = submission
     @disability = submission.form.dig('form526', 'form526', 'disabilities')&.first
     @disability_code = @disability['diagnosticCode'] if @disability
-    @rrd_status = RapidReadyForDecision::Form526BaseJob.rrd_status(submission)
+    @rrd_status = submission.rrd_status
```

Comment:
If desired, we could get rid of this instance variable and call `submission.rrd_status` directly.